src/tasks/withdraw.ts

```diff
@@ -19,18 +19,28 @@ const ONEINCH_ETH_FLAG = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
 const DAI_DECIMALS = 18;
 
 interface Withdrawal {
-  token: TokenDetails;
+  token: PricedToken;
+  amount: BigNumber;
+  amountUsd: BigNumber;
   balance: BigNumber;
-  usdValue: BigNumber;
+  balanceUsd: BigNumber;
 }
 
 interface DisplayWithdrawal {
   symbol: string;
   balance: string;
+  amount: string;
   value: string;
   address: string;
 }
 
+interface PricedToken extends TokenDetails {
+  // The number of wei needed to form a unit of this token (10**decimals)
+  oneInWei: BigNumber;
```

Comment:
wouldn't a dynamic method that computes this from decimals be simpler (also because it looks like it's only used once)?

Reply:
Mine was a workaround because the number of decimals could be unknown, and I didn't want to set some default value at many different places in the code in case the guess would become inconsistent with time.
The new code enforces the guess of the decimals when creating `PricedToken`, I definitely like more the result now.